projects/media-viewer/src/lib/store/selectors/bookmarks.selectors.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { TestBed } from '@angular/core/testing';
+import { select, Store, StoreModule } from '@ngrx/store';
+import { reducers, State } from '../reducers/reducers';
+
+import * as fromSelectors from './bookmarks.selectors';
+import * as fromDocActions from '../actions/document.action';
+import * as fromActions from '../actions/bookmarks.action';
+
+describe('Bookmark selectors', () => {
+  let store: Store<State>;
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({}),
+        StoreModule.forFeature('media-viewer', reducers),
+      ],
+    });
+    store = TestBed.get(Store);
+    spyOn(store, 'dispatch').and.callThrough();
+  });
+
+  describe('getBookmarksPerPage', () => {
+    it('should return pages of bookmarks', () => {
+      store.dispatch(new fromDocActions.AddPages([{
+        div: {},
+        scale: 1,
+        rotation: 0,
+        id: '1',
+        viewportScale: 1.33333
+      }]));
+
+      let result;
+      store.pipe(select(fromSelectors.getBookmarksPerPage)).subscribe(value => {
+        result = value;
+      });
+      const payload: any = {
+        id: '1',
+        documentId: 'uuid',
+        name: 'bookmark1',
+        pageNumber: 0,
+        xCoordinate: 100,
+        yCoordinate: -120
+      };
+      store.dispatch(new fromActions.CreateBookmark(payload));
```

Comment:
do we need the CreateBookmark action dispatch here, since the test is for the selector? I think the action dispatch needed is the AddPages one, which we're already doing above